### PR TITLE
feat: support korean

### PR DESCRIPTION
**Pull request by WayneKim92**
Support Korea

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "ava": "5.2.0",
     "c8": "7.13.0",
     "coveralls": "3.1.1",
-    "eslint": "8.36.0",
+    "eslint": "8.22.0",
```
Comment (dargmuesli) [inline package.json:42]: Why downgrade eslint?

**Comment by dargmuesli** [comment]
Thank you for your pull request! :heart:

**Comment by WayneKim92** [comment]
I had the same issue with the link below.

https://stackoverflow.com/questions/73509984/eslint-typeerror-this-liboptions-parse-is-not-a-function

**Comment by dargmuesli** [comment]
I don't think that specific ide issues should change to codebase this way.

Published as v0.7.0 :tada:

**Pull request merged**